apps/clock/style/alarm.css

```diff
@@ -75,6 +75,10 @@ ul.compact li label:active {
   display: none;
 }
 
+#time-select {
+  text-indent: 1000rem;
```

Comment:
what about using a different unit instead of `rem`? that way the hack would work even if the field is wider than `1000rem` (bigger devices, 4K, etc..) - I did a test and `110%` seems to work